### ISSUE TITLE
Use ES6 module

### ISSUE THREAD

**Issue by mjeanroy** [sapegin/jquery.mosaicflow#58]
Hi @sapegin,

I would like to discuss with you if you were interested for a migration to use ES6 module on this project. I know that this project is not actively maintain, but I would be happy to work on it and to send you a PR for that. I think this migration could be great as we will be able to use the library with an ES6 bundler (rollup or webpack) without any plugins.

What do you think?

**Comment by sapegin** [comment]
Hi @mjeanroy!

I don’t really understand how it would help with bundlers. Are you talking about the `module` filed in `package.json`?

Anyway I’ll be happy to transfer the repo or give you commit access. I haven’t used jQuery in last three years I think ;-)

**Comment by mjeanroy** [comment]
> I don’t really understand how it would help with bundlers. Are you talking about the module filed in package.json?

Exactly. I am migrating an old application to use rollup, and if I want to keep your library, I need to use `rollup-plugin-commonjs` or `rollup-plugin-amd`. That said, I also need one of these plugins because of jQuery, but I think it should not be so hard to migrate this library.

> I haven’t used jQuery in last three years I think ;-)

You are lucky :)
